page-impact.hbs

```diff
@@ -0,0 +1,134 @@
+            <div justify="center" align-items="center" class="elfsight-app-7a9bc7c0-706f-48a4-8180-3f6e65f1f058"></div>
+
+        </div>
+    </main>
+</div>-->
+
+<section class="text-gray-600 body-font">
+    <div class="container px-5 py-24 mx-auto">
+        <div class="flex flex-col text-center w-full">
+            <h1 class="entry-title">IMPACT</h1>
+            <br>
+            <h1 class="single-title">ONLINE REPOSITORY</h1>
+            <center><img src="https://i0.wp.com/advisory.sg/wp-content/uploads/elementor/thumbs/Online-Repository-oho6i9yghlfxqk80hycs3ligynbxh7r1fcnxhlmlmo.png?w=800&amp;ssl=1" title="Online Repository" alt="Online Repository"><center>
+            <br>
+            <center><p class="entry-title"><font size="4.5" color="#696969">From <strong>November 2017</strong> to <strong>January 2021</strong>, Advisory's <strong>online repository</strong> has had:<br></font><br></p><center>
+        </div>
+
+        <div class="container-impact-number">
+            <div class="xl:w-1/4 lg:w-1/2 md:w-full px-8 py-6"><center>
+                <h1 class="single-title">235,576</h1>
+                <p class="entry-title"><font size="4.5" color="#808080">Total Repository Views</font></p>
+            </center></div>
+            <div class="xl:w-1/4 lg:w-1/2 md:w-full px-8 py-6"><center>
+                <h1 class="single-title">105,216</h1>
+                <p class="entry-title"><font size="4.5" color="#808080">Users</font></p>
+            </center></div>
+            <div class="xl:w-1/4 lg:w-1/2 md:w-full px-8 py-6"><center>
+                <h1 class="single-title">7,853</h1>
+                <p class="entry-title"><font size="4.5" color="#808080">Average Views Per Month</font></p>
+            </center></div>
+            <div class="xl:w-1/4 lg:w-1/2 md:w-full px-8 py-6"><center>
+                <h1 class="single-title">129</h1>
+                <p class="entry-title"><font size="4.5" color="#808080">Published Articles</font></p>
+            </center></div>
+        </div>
+
+        <br>
+        <div class="flex flex-col text-center w-full">
+            <h1 class="single-title">OFFLINE PROGRAMMING</h1>
+        </div>
+
+        <br>
+        <div class="container-impact-picture">
+            <div> <!-- width="800" height="611" was original setting-->
+                <img width="320" height="244" src="https://i0.wp.com/advisory.sg/wp-content/uploads/2019/12/D-Space.jpg?resize=1024%2C782&ssl=1">
+            </div>
+            <div>
+                <img width="320" height="244" src="https://i0.wp.com/advisory.sg/wp-content/uploads/elementor/thumbs/Centre-image-ohwc21w9k9l2g53u7lvundxq8fp75yhy3v3wybo40e.jpg?w=800&ssl=1">
+            </div>
+            <div>
+                <img width="320" height="244" src="https://i0.wp.com/advisory.sg/wp-content/uploads/elementor/thumbs/Explore-Google-ohwbecet6p4pm7jkncylrdb6ukjx0sdo6ir172tl18.jpg?w=800&ssl=1">
+            </div>
+        </div> 

```

Comment:
Please restrict the width of this part so that the numbers don't spread out as much.